docs/execplans/1-4-15-4-windows-ci-on-x64.md

```
@@ -0,0 +1,603 @@
+git status --short --branch
+nl -ba .github/workflows/nodejs.yml | sed -n '1,280p'
+```
+
+Expected baseline signal:
+
+```plaintext
+Current branch is the task branch; workflow shows windows-latest in matrix and
+no Windows aarch64 lane.
+```
+
+2. Recommended CI evidence collection (if `gh` is available):
```

markdownlint-cli2 (0.21.0)

[warning] Ordered list item prefix
Expected: 1; Actual: 2; Style: 1/1/1

(MD029, ol-prefix)